.github/workflows/release.yml

```diff
@@ -164,11 +164,51 @@ jobs:
           docker push gluwa/creditcoin:latest
           docker logout
 
+  build-creditcoin-js:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - run: npm install -g yarn
+
+      - name: Figure out tag name
+        shell: bash
+        run: |
+          TAG_NAME=$(git describe --tag)
+          echo "TAG_NAME=$TAG_NAME" >> $GITHUB_ENV
+
+      # TODO: should we Auto-update typedefs ????
```

Comment:
@nathanwhit, @CAGS295 I don't have a strong opinion here. For integration tests we always rebuild the typedefs.

Reply:
I say we should.

Reply:
Turns out we're already doing this on every PR for the `dev` branch, see https://github.com/gluwa/creditcoin/blob/dev/.github/workflows/ci.yml#L266

Looks like the last time this happened was in https://github.com/gluwa/creditcoin/pull/579. I've added a commit to see `git diff` b/c ATM I'm not sure if we aren't missing something.

Reply:
No updates for typedefs -> https://github.com/gluwa/creditcoin/actions/runs/3339614124/jobs/5528776536. 

Looks like we're good.